woocommerce/class-sv-wc-plugin.php

```diff
@@ -183,6 +183,26 @@ public function __construct( string $id, string $version, array $args = [] ) {
 		$this->add_hooks();
 	}
 
+	/**
+	 * Provides backward compatibility for arguments, where we can. This handles any format changes in the $args array.
+	 *
+	 * @param array $args
+	 * @return array
+	 */
+	protected function maybeHandleBackwardsCompatibleArgs(array $args): array
+	{
+		// handle format change for HPOS declaration
+		if (
+			array_key_exists('supports_hpos', $args) &&
+			! isset($args['supported_features']['hpos'])
+		) {
+			$args['supported_features']['hpos'] = $args['supports_hpos'];
+			unset($args['supports_hpos']);
+		}
```

Comment:
How about we unset the `supports_hpos` anyways without checking if `supported_features.hpos` is set? As well as making sure `supported_features` key is initialized

Reply:
Aka [f3c9a7e](https://github.com/godaddy-wordpress/wc-plugin-framework/pull/719/commits/f3c9a7e1a0c20e825b604c7a5b8ba6cd233046b6)

Reply:
LGTM @nmolham-godaddy ! Thank you!